packages/blockchain-wallet-v4-frontend/src/modals/Btc/SendBtc/FirstStep/index.js

```diff
@@ -15,7 +15,6 @@ class FirstStep extends React.Component {
 
   render () {
     const { data, actions } = this.props
-
     return data.cata({
       Success: value => (
         <Success
```

Comment:
Maybe a paypro ? PayProSuccess : Success template would be better, instead of all the logic in template.success.js.

Reply:
I was thinking that too, guessing you mean like a new PayProSuccess template? I can do that but was thinking I'd be repeating a lot of code that's already in template.success.js (Success) if I'm understanding you correctly, figured it might be better to just add a couple tidbits of paypro specific stuff inside the existing template instead of creating a whole new first step template with much of the same stuff thats in Success

Reply:
Yeah, the more I think about it the more I think I like you're approach more.